src/Http/Http.Extensions/src/HttpResponseJsonExtensions.cs

```diff
@@ -212,8 +213,7 @@ static async Task WriteAsJsonAsyncSlow(HttpResponse response, object? value, Jso
     {
         ArgumentNullException.ThrowIfNull(response);
 
-        var options = ResolveSerializerOptions(response.HttpContext);
-        return response.WriteAsJsonAsync(value, jsonTypeInfo: options.GetTypeInfo(type), contentType: null, cancellationToken);
+        return response.WriteAsJsonAsync(value, type, options: null,  contentType: null, cancellationToken);
```

Comment:
Does the null parameter mean that it's using the default options or that the options instance is being resolved elsewhere?

Reply:
It means it is using the options instance that is resolved from DI, if available. If not, it uses the JsonOptions.DefaultSerializerOptions. See:


https://github.com/dotnet/aspnetcore/blob/d07f377b6c4c72f44a0eb2810c8abf1ae3fabd40/src/Http/Http.Extensions/src/HttpResponseJsonExtensions.cs#L254-L275

https://github.com/dotnet/aspnetcore/blob/d07f377b6c4c72f44a0eb2810c8abf1ae3fabd40/src/Http/Http.Extensions/src/HttpResponseJsonExtensions.cs#L339-L343